pyof/foundation/basic_types.py

```diff
@@ -627,13 +651,15 @@ def insert(self, index, item):
         Raises:
             :exc:`~.exceptions.WrongListItemType`: If the item has a different
                 type than the one specified in the constructor.
+
         """
         if issubclass(item.__class__, self._pyof_class):
             list.insert(self, index, item)
         else:
             raise exceptions.WrongListItemType(item.__class__.__name__,
                                                self._pyof_class.__name__)
 
+    # pylint: disable-msg=arguments-differ
```

Comment:
This should not affect the rest of the class. Either write in the same lame as `def` (recommended) or enable it right after the method.